admin-manual/maintenance/maintenance.rst

```diff
@@ -134,72 +142,280 @@ The index names are: `aips`, `aipfiles`, `transfers` and `transferfiles`.
+
+
+
 .. _admin-faq:
 
 FAQ
 ---
 
-.. _restart-services:
+.. _disk-full:
 
-How to restart the Archivematica services
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How to clean up a full disk
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    "My Archivematica disk filled up and now Archivematica won't work. How can I
+    fix this?"
 
-**Restart all services**
+Archivematica servers have as much storage as they have been commissioned. If
+processing lots of very large files, particularly if working with normalization,
+this will cause the disk to fill up and cause the system to malfunction.
+
+When the disk on an Archivematica instance is full, a number of steps need to be
+taken to recover. 
+
+**Recovery protocol**
+
+#. Clean up the disk by removing failed or rejected transfers, any excessive
+   /tmp data, or anything else causing the disk to have filled up.
+#. Reset MySQL (or MariaDB, on CentOS) database.
+#. Reset Archivematica components in appropriate order (see `restart-services`_
+   for details).
+#. Set Elasticsearch back into write mode. The easiest way to do this is to run
+   the following command:
 
 .. code:: bash
 
-   am services
+    curl -XPUT -H "Content-Type: application/json" http://localhost:9200/_all/_settings -d '{"index.blocks.read_only_allow_delete":null}'
+
+
+.. _restart-services:
 
-Note that the default action is to restart all services. To see other available
-parameters, type:
+How to restart services
+^^^^^^^^^^^^^^^^^^^^^^^
+    "Something is not working right, or I need to stop a hanging transfer. What
+    can I do?"
+
+Archivematica is made up of these four core components:
 
 .. code:: bash
 
-   am services help
+    archivematica-mcp-server
+    archivematica-mcp-client
+    archivematica-dashboard
+    archivematica-storage-service
+
+Other services that Archivematica depends on are:
+  * ClamAV
+  * ElasticSearch
+  * Gearman
+  * MySQL (Ubuntu) or MariaDB (CentOS)
+  * Nailgun
+  * Nginx
 
-**Stopping**
+Each service can be started/stopped/restarted with:
 
 .. code:: bash
 
-   sudo stop archivematica-mcp-server
-   sudo stop archivematica-mcp-client
-   sudo /etc/init.d/apache2 stop
-   sudo /etc/init.d/gearman-job-server stop
-   sudo stop mysql
-   sudo /etc/init.d/elasticsearch stop
+    service <name> start|stop|status|restart
 
-**Starting**
+To restart all services, restart the Gearman service and each Archivematica
+component, in this order:
 
 .. code:: bash
 
-   sudo /etc/init.d/elasticsearch start
-   sudo start mysql
-   sudo /etc/init.d/gearman-job-server start
-   sudo /etc/init.d/apache2 start
-   sudo start archivematica-mcp-server
-   sudo start archivematica-mcp-client
+    service gearmand restart
+    service archivematica-mcp-server restart
+    service archivematica-mcp-client restart
+    service archivematica-dashboard restart
+    service archivematica-storage-service restart
+
+**Note: Depending on your installation, gearmand might be called gearman-job-server.**
```

Comment:
You could format this as an admonition:

```
.. note::

   Depending on your installation...
```

Then it will get some nice formatting to hightlight it.

Reply:
Thank you! I changed it in two places.